app/controllers/admins_controller.rb

```diff
@@ -0,0 +1,32 @@
+class AdminsController < ApplicationController
+  def index
+    admin = Admin.all
+    render json: admin, status: :ok
+  end
+
+  def create
+    admin = Admin.create!(admin_params)
+    render json: admin, status: :created
+  end
+
+  def update
+    admin = Admin.update(admin_params)
```

Comment:
Use a find private method to find the user to update. 
Use update without exclamation for now

Reply:
reviewed and implemented changes